### ISSUE TITLE
Abstract function traslation from sympy to sage

### ISSUE THREAD

**Issue by man74cio** [sympy/sympy#12825]
Abstract function translation sympy -> sage is not well implemented :  
In the case of known functions (like sin, cos, exp) the translation is ok but for abstract functions 
the translation fails (test from sage):

```python
from sympy import *
x,y = symbols('x y')
f = Function('f')
expr =  f(x,y)
expr._sage()
```
gives:
```python
AttributeError                            Traceback (most recent call last)
<ipython-input-67-995abf46eb43> in <module>()
----> 1 a._sage_()

/home/mmancini/Sage-last/sage/local/lib/python2.7/site-packages/sympy/core/function.pyc in _sage_(self)
    705         import sage.all as sage
    706         fname = self.func.__name__
--> 707         func = getattr(sage, fname)
    708         args = [arg._sage_() for arg in self.args]
    709         return func(*args)

AttributeError: 'module' object has no attribute 'f'
```

Proposed changes in sympy/core/function.py : 
Replace the function FunctionClass._sage_ 

 ```python
def _sage_(self):
        import sage.all as sage
        fname = self.func.__name__
        func = getattr(sage, fname)
        args = [arg._sage_() for arg in self.args]
        return func(*args)
```
with

```python
def _sage_(self):
        import sage.all as sage
        fname = self.func.__name__
        func = getattr(sage, fname,None)
        args = [arg._sage_() for arg in self.args]
        if func is None:
            # abstract function
            return sage.function(fname)(*args)

        return func(*args)


```

**Comment by mkoeppe** [comment]
This was apparently fixed in #12826; and it works correctly in current Sage. Should close

**Comment by oscarbenjamin** [comment]
Thanks
